tests/test_minimal/test_tools/aws_tools.py

```diff
@@ -1,10 +1,13 @@
+import datetime
 import os
 import time
 
 import boto3
 
 from neuroconv.tools.aws import submit_aws_batch_job
 
+_RETRY_STATES = ["RUNNABLE", "PENDING", "STARTING", "RUNNING"]
```

Comment:
This is one of those bug fixes I mentioned sneaking into this PR - after running the tests so many times I did see some edge cases show up in the waiting times

I think a final follow-up in the series (once everything works together) could be more elegant with some common utils for such things